piksi_multi_rtk_ros/install/install_piksi_multi.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 GIT_REPO_LIBSBP=https://github.com/swift-nav/libsbp.git
-REPO_TAG=v2.2.15 #version you want to checkout before installing
+REPO_TAG=v2.3.11 #version you want to checkout before installing
```

Comment:
Why do you want to change this in master? Master has been tested with SBP 2.2.15, as written here https://github.com/ethz-asl/ethz_piksi_ros/tree/master/piksi_multi_rtk_ros#firmware-and-sbp-lib-version

I'm working on another pull request to switch to 2.3.10: https://github.com/ethz-asl/ethz_piksi_ros/pull/16

Reply:
I had been working against 2.3.11 (which was the newest tag of SBP at the time) but 2.3.10 would be fine as well. I will adjust the PR.

Reply:
We just deiced to stick to the indication here: [Piksi Multi and Duro SBP Specifications](https://support.swiftnav.com/customer/en/portal/articles/2492810-swift-binary-protocol).

I'd like to test my PR today and merge it asap ;)